<ui version="4.0" >
 <class>Kleo::NewCertificateUi::ResultPage</class>
 <widget class="QWizardPage" name="Kleo::NewCertificateUi::ResultPage" >
  <property name="title" >
   <string comment="@title" >Key Pair Successfully Created</string>
  </property>
  <property name="subTitle" >
   <string>Your new key pair was created successfully. Please find below details on the result and some suggested next steps.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="resultGB" >
     <property name="title" >
      <string>Result</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QTextBrowser" name="resultTB" />
      </item>
      <item>
       <widget class="QTextBrowser" name="errorTB" />
      </item>
      <item>
       <widget class="Kleo::DragQueen" name="dragQueen" >
        <property name="toolTip" >
         <string>Drag this icon to your mail application's composer to attach the request to a mail.</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="restartWizardPB" >
     <property name="text" >
      <string>Restart This Wizard (Keeps Your Settings)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="nextStepsGB" >
     <property name="title" >
      <string>Next Steps</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QPushButton" name="saveRequestToFilePB" >
        <property name="text" >
         <string>Save Request To File...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendRequestByEMailPB" >
        <property name="text" >
         <string>Send Request By EMail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="makeBackupPB" >
        <property name="text" >
         <string>Make a Backup Of Your Certificate...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendCertificateByEMailPB" >
        <property name="text" >
         <string>Send Certificate By EMail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="uploadToKeyserverPB" >
        <property name="text" >
         <string>Upload Certificate To Directory Service...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createRevocationRequestPB" >
        <property name="text" >
         <string>Create a Revocation Request...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kleo::DragQueen</class>
   <extends>QLabel</extends>
   <header>utils/dragqueen.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>saveRequestToFilePB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ResultPage</receiver>
   <slot>slotSaveRequestToFile()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>34</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel" >
     <x>2</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendRequestByEMailPB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ResultPage</receiver>
   <slot>slotSendRequestByEMail()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>32</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel" >
     <x>6</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendCertificateByEMailPB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ResultPage</receiver>
   <slot>slotSendCertificateByEMail()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>28</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel" >
     <x>4</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadToKeyserverPB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ResultPage</receiver>
   <slot>slotUploadCertificateToDirectoryServer()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>30</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel" >
     <x>2</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>makeBackupPB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ResultPage</receiver>
   <slot>slotBackupCertificate()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>32</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel" >
     <x>4</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createRevocationRequestPB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ResultPage</receiver>
   <slot>slotCreateRevocationRequest()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>32</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel" >
     <x>2</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSaveRequestToFile()</slot>
  <slot>slotSendRequestByEMail()</slot>
  <slot>slotSendCertificateByEMail()</slot>
  <slot>slotUploadCertificateToDirectoryServer()</slot>
  <slot>slotBackupCertificate()</slot>
  <slot>slotCreateRevocationRequest()</slot>
 </slots>
</ui>
